cfgov/ask_cfpb/models/answer_page.py

```diff
@@ -186,7 +186,17 @@ class AnswerPage(CFGOVPage):
         help_text="Include share and print buttons above answer."
     )
 
+    # TODO: When we've updated to Wagtail 2.13.x remove the help_text
+    # and set the block_count for the notification to 1.
+    # See https://git.io/JODqS
+    notification = StreamField([
+        ('notification', molecules.Notification(
+            help_text="Include only one notification."))],
+        blank=True,
+    )
+
     content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('notification'),
```

Comment:
It might make sense to move this to just above the answer content panel, given the order the content is output on the page.

Reply:
Updated https://github.com/cfpb/consumerfinance.gov/pull/6412/commits/6147cdc242f59a98a2f336246af1ea916d5a5ade